.changeset/strange-rocks-poke.md

```diff
@@ -0,0 +1,5 @@
+---
+"hardhat": patch
+---
+
+Fix - allow recovering from partially downloaded compiler list
```

Comment:
These messages end up in the changelong and other user-facing places, so we don't use any kind of structured message, but rather a plain English explanation. Something like 

> Fix a bug that prevented Hardhat from working if the compilers list was partially downloaded